x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/150_lookup.yml

```diff
@@ -38,7 +38,7 @@ basic:
         - method: POST
           path: /_query
           parameters: []
-          capabilities: [lookup_command]
+          capabilities: [lookup_command, tables_types]
```

Comment:
Hm, `tables_types` implies `lookup_command`, so maybe we can just replace the capability instead of adding to the list?

Reply:
I'm not sure if it's clearer to put both or only the latest one.